test/framework/build.gradle

```diff
@@ -73,3 +75,7 @@ test.configure {
   systemProperty 'tests.gradle_wire_compat_versions', bwcVersions.wireCompatible.join(',')
   systemProperty 'tests.gradle_unreleased_versions', bwcVersions.unreleased.join(',')
 }
+
+task integTest(type: RandomizedTestingTask) {
```

Comment:
I'd probably rename the test to match the normal suffix.

Reply:
I mean, instead of adding the new test run, I'd fix the test's name to match the normal "Tests" suffix.

Reply:
There's just a few of these tests, but they all extend `ESIntegTestCase` . I think that calling them `Tests` rather than `IT` would go against  the naming convention.

Reply:
It depends on where they are in the project. If they were in the server, yes. Otherwise, we do tend to call the `Tests`. The naming convention isn't super consistent, sadly.